R/pages.R

```diff
@@ -25,3 +25,31 @@ page404 <- function(page = NULL, message404 = NULL){
     return(page)
   }
 }
+
+#' Fix conflicts when some bookmark uses bootstrap
+#'
+#' @description This function dynamically removes bootstrap dependency when user opens specified bookmark.
```

Comment:
I think that `@description` here is redundant as everything below the title and without tag is taken as a description.